iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/module/AbstractBuilderModule.java

```diff
@@ -70,7 +68,7 @@ protected DataInfo buildInference(float confidenceLevel) throws TrustLevelThresh
      * @throws TrustLevelThresholdExceededException thrown when trust level threshold was exceeded
      */
     protected DataInfo buildInference(float confidenceLevel, String inferenceProvenance) throws TrustLevelThresholdExceededException {
-        float currentTrustLevel = confidenceLevel * InfoSpaceConstants.CONFIDENCE_TO_TRUST_LEVEL_FACTOR;
+        float currentTrustLevel = new ConfidenceLevelConverter().convertToTrustLevel(confidenceLevel);
```

Comment:
I'd go with static method here and in all similar cases. 

If we agree to rely on a static conversion factor we won't need to instantiate  the immutable object for such a simple task.